fastlane/metadata/android/en-US/changelogs/123.txt

```diff
@@ -0,0 +1,5 @@
+* Add themed launcher Icons.
+* Bump minSDK to 21.
```

Comment:
Would something like a minimum required android version be easier to understand for end-users?

Reply:
My plan is to let @Echolon take care of the change log in #854 and I'll copy & paste here. :slightly_smiling_face: 
But if that process doesn't work out, I will just gonna make the changes here.